src/components/SEO/Default.tsx

```diff
@@ -28,7 +28,7 @@ const SEOHeader = ({ router }: SEOHeaderProps) => (
     twitter={{
       cardType: 'summary_large_image',
       handle: '@Everipedia',
-      site: 'Everipedia',
+      site: 'BrainDAO',
```

Comment:
https://developer.twitter.com/en/docs/twitter-for-websites/cards/overview/summary

site btw should be a twitter handle, so probably @everipedia as well

Reply:
wow 😲

Reply:
old was wrong too I think it needs the @ (but didnt test it)